openshift/templates/base-images/mssql-2019.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Template
```

Comment:
I'm curious why this is needed too?  Doesn't this image already exist in our image streams?

Reply:
See my earlier comment

Reply:
thanks for doing this Alejandro!